pkg/utils/vmi.go

```diff
@@ -49,3 +53,15 @@ func (v *VmiGetter) WhoUseNad(nad *nadv1.NetworkAttachmentDefinition, nodesFilte
 
 	return afterFilter, nil
 }
+
+func vmiByNetwork(obj *kubevirtv1.VirtualMachineInstance) ([]string, error) {
```

Comment:
Do we want to align the naming between the label and the function to both `vmi`?